### ISSUE TITLE
parseYAML fails instead of returning a valid object/list of objects

### ISSUE THREAD

**Issue by mplzik** [grafana/tanka#194]
When trying to load a YAML using parseYaml from tanka source code using following snippet:
```jsonnet
{
  local parseYaml = std.native('parseYaml'),
  local crds_yaml = parseYaml(importstr 'external.yaml'),

  a: crds_yaml
}

```
(`external.yaml` contains `a: 47`)

Tanka fails with following error:
```
evaluating jsonnet: RUNTIME ERROR: json: unsupported type: map[interface {}]interface {}
	main.jsonnet:3:21-57	object <anonymous>
	main.jsonnet:5:6-15	object <anonymous>
	During manifestation	
```

**Comment by mplzik** [comment]
The current master (1ba29d4933da7828873523e53fc6e1a0d6fd281b) returns:
```
evaluating jsonnet: unexpected end of JSON input
```
instead, which is also a bit weird.

**Comment by mplzik** [comment]
https://github.com/grafana/tanka/pull/195

**Comment by sh0rez** [comment]
This is fixed in #195
